Gopkg.toml

```diff
@@ -42,6 +42,11 @@ required = [
   name = "k8s.io/client-go"
   version = "kubernetes-1.11.0"
 
+[[override]]
```

Comment:
There are 2 versions of pkg dependency in toml file. I think you can update the one under `constraint` to recent sha.

Reply:
oh :sweat_smile: I didn't look below the `[[prune]]` options, my bad :sweat_smile: